testworld/proof_test.go

```diff
@@ -0,0 +1,38 @@
+// +build testworld
+
+package testworld
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestProofWithMultipleFields_successful(t *testing.T) {
+	alice := doctorFord.getHostTestSuite(t, "Alice")
+	bob := doctorFord.getHostTestSuite(t, "Bob")
+
+	// Alice shares invoice document with Bob
+	res, err := alice.host.createInvoice(alice.httpExpect, http.StatusOK, defaultNFTPayload([]string{bob.id.String()}))
+	if err != nil {
+		t.Error(err)
+	}
+
+	docIdentifier := getDocumentIdentifier(t, res)
+	if docIdentifier == "" {
+		t.Error("docIdentifier empty")
+	}
+
+	// Alice want's to get a proof
+	proofPayload := map[string]interface{}{
```

Comment:
please move this to the payloads.go you created.
also get a proof from bobs node and match with Alices.